news/2024/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area.md

```diff
@@ -0,0 +1,307 @@
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+If there's an ELO matchmaking system added later, would you grind it?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3717598" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-xootynator.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>
+
+Maybe! I know there's been systems like that before like Maidbot which I used and was pretty cool at the time, but I'm not sure how much incentive it has to get people to grind it. There's only so many times I can play the same pools over and over again at least... but there's actually a pretty cool osu! ELO system I heard of called the osu! Performance Ranking! (please don't silence me)
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+What other games besides osu! have you played the most?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3717598" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-xootynator.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>
+
+I think the games I have with the most hours are Archeage and Smite (unfortunately). I used to be a pretty big MMO player when I was younger, starting with Maplestory. Not really grinding or anything but just chill progression and exploring. That's also why one of my favourite games of all time has been Skyrim, which I've been playing since its release! I think I played Smite heavily for about two years, again not really competitively but I liked the gameplay a lot. I have pretty much never been competitive in a game besides osu!, probably because I can't stand random people yelling at me U_U
```

Comment:
```suggestion
I think the games I have with the most hours are Archeage and Smite (unfortunately). I used to be a pretty big MMO player when I was younger, starting with Maplestory. Not really grinding or anything but just chill progression and exploring. That's also why one of my favourite games of all time has been Skyrim, which I've been playing since its release!

I think I played Smite heavily for about two years, again not really competitively, but I liked the gameplay a lot. I have pretty much never been competitive in a game besides osu!, probably because I can't stand random people yelling at me U_U
```